### PR TITLE
[FIX] AV-3975: Frames in grid view

### PR DESCRIPTION
**Pull request by YasheshSavani**
Ticket: https://20bn.atlassian.net/browse/AV-3975

Description:
This PR will fix the issue of scrolling sideways for some of the videos whose frames were coming in a single row.

### CODE REVIEW

File: scripts/tools/annotation/templates/up_liste.html [hunk 1]
```diff
@@ -10,52 +10,46 @@ <h2>Video: {{video_name}}</h2>
...
-        {% for imgs in images %}
-           <div class="row">
-            <table>
-                <tr>
+        <ul class="fichiers" style="padding: 0px;" >
+            {% for imgs in images %}
+            <div class="row" style="display: flex;">
                 {% for img, num, pred_class in imgs %}
-                    <td>
+                <div class="col" style="margin:10px">
```
Comment (antoinemrcr) [inline scripts/tools/annotation/templates/up_liste.html:17]: I think we have css files. can we use that ?

Comment (YasheshSavani) [reply scripts/tools/annotation/templates/up_liste.html:17]: yes, it was just for the testing purpose, now I'll clean the code for final push.

Comment (YasheshSavani) [reply scripts/tools/annotation/templates/up_liste.html:17]: Thank you though

File: scripts/tools/annotation/annotation.py [hunk 2]
```diff
@@ -76,14 +76,21 @@ def annot(nom):
...
     chunk_size = 5
-    n_chunk = int(len(images)/chunk_size)
-    images = np.array_split(images, n_chunk)
+    n_chunk = int(len(_images) / chunk_size)
+    remaining_images = len(_images) % chunk_size
+    if remaining_images:
+        images = np.array_split(_images[:-remaining_images], n_chunk)
+        images.append(np.array(_images[-remaining_images:]))
+    else:
+        images = np.array_split(_images, n_chunk)
```
Comment (sunny-panchal) [inline scripts/tools/annotation/annotation.py:93]:
```suggestion
images = np.split(images, np.arange(chunk_size, len(images), chunk_size))
```

I think this should do the trick too ^ (after renaming `_images` to `images`

**Review by sunny-panchal — COMMENTED** [review]
Haven't been able to test because I get a "404 Page not found" when I try to open the tool but here are some general comments

File: scripts/tools/annotation/prepare_annotation.py [hunk 3]
```diff
@@ -16,9 +16,9 @@
 
 from docopt import docopt
 
-from realtimenet import feature_extractors
-from realtimenet import engine
-from realtimenet.finetuning import compute_features
+from sense import feature_extractors
```
[the remark this answers is not in the document]
Comment (YasheshSavani) [reply scripts/tools/annotation/prepare_annotation.py:19]: They are not fixed in annotation branch yet.

**Review by sunny-panchal — APPROVED** [review]
looks good :+1:

**Pull request merged**